applications/asset_tracker/Kconfig

```diff
@@ -286,6 +286,18 @@ config CLOUD_CONNECT_ERR_REBOOT_S
 
 endmenu # Cloud
 
+menu "nRF Connect for Cloud"
+
+config NRFCLOUD_CLIENT_ID_PREFIX
```

Comment:
This config should not be defined in the `asset_tracker` given it's used directly by the `nrf_cloud` library. I'd rather see it in https://github.com/nrfconnect/sdk-nrf/blob/master/subsys/net/lib/nrf_cloud/Kconfig.

Reply:
Makes sense!